electron-app/src/components/helper/TabNavigation.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <v-tabs
+    fixed-tabs
+    height="36px"
+    hide-slider
+    active-class="rounded-tabs__active"
```

Comment:
since active is a modifier can you prefix it with `--` instead? http://getbem.com/
Also, it would be nice if we follow the pattern of re-using the component name in the styling as with the other components

Reply:
Sure. So something like `active-class="tab-navigation__tab--active"` ?